README.md

```diff
@@ -20,8 +20,12 @@ This repository contains Docker configuration aimed at Moodle developers and tes
 ## Quick start
 
 ```bash
+# Get Moodle code (or get another version, or skip if you have a local copy)
+git clone -b MOODLE_403_STABLE git://git.moodle.org/moodle.git
```

Comment:
I'd suggest to use `/path/to/moodle/code` here and then leave the export below unmodified.

```suggestion
git clone -b MOODLE_403_STABLE git://git.moodle.org/moodle.git /path/to/moodle/code
```

Reply:
That means that copy-pasting the code no longer works. Additional steps are required.

For a "quick start" section, it could be a best practice that minimal brain power is required to proceed, and any further customization could be explained but not required at that time.

Reply:
Heh,

`/path/to/moodle/code` doesn't look (to me) like requiring too much brain power. Whoever wants to use "moodle" (or "/tmp/moodle" or "C:\\Users\meow\moodle403" or anything else) can do it perfectly.

And omitting it makes the quick-start less obvious for people (the majority maybe? At least that's my case) that already have a clone/checkout/codebase available, somewhere (@ `/path/to/moodle/code` exactly) and don't need to run that new, optional, command.

Sorry for not agreeing, but that's how I see it. And thanks for the PR!

Ciao :-)

Reply:
Thank you. I do see that specifying the output location is a net win.

And luckily we already know that. So here is some progress: https://github.com/moodlehq/moodle-docker/pull/286/commits/e4ce4e52be8a96714dd40a7f0e5f9410b3cb76c0